src/app/pages/loading-waiting-states/contextual-spinners/contextual-spinners.component.scss

```diff
@@ -1,19 +1,87 @@
 @import '~@pxblue/colors/palette.scss';
+$duration: 200ms;
+$animationTimingFunction: ease-out;
+section.mat-typography {
+    padding: 16px 24px;
+    .example-section {
+        margin-bottom: 16px;
+    }
+    .show-text {
+        .text-container {
+            display: block;
+            -webkit-animation: fadeInAnimation $duration ease-out;
+            -moz-animation: fadeInAnimation $duration ease-out;
+            -o-animation: fadeInAnimation $duration ease-out;
+            animation: fadeInAnimation $duration ease-out;
+        }
+        .spinner-container {
+            display: none;
+        }
+    }
+
+    .raised-button-example {
+        height: 36px;
+        .text-container {
+            display: none;
+        }
+        .spinner-container {
+            display: block;
+            margin: 0 auto;
+            -webkit-animation: fadeInAnimation $duration ease-out;
+            -moz-animation: fadeInAnimation $duration ease-out;
+            -o-animation: fadeInAnimation $duration ease-out;
+            animation: fadeInAnimation $duration ease-out;
+        }
+    }
+
+    .extended-fab-button {
+        width: 179px;
```

Comment:
why did we change this from 180?

Reply:
As per figma button width is 179px. So changed to 179px.
![image](https://user-images.githubusercontent.com/10433274/123414771-ef069800-d5d1-11eb-9cb6-dce037e13dd6.png)

Reply:
Since we cannot do `width:auto` for animations, let's change the button width to 166px. I think Figma's button is using a slightly different typography as this, and to compensate for that, 166px is more balanced.